commit 4b79840af6b4fddaef412826087fc6ea354d0a33 (refs/changes/85/1885/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-05-02T13:52:04-07:00 (2 years, 5 months ago)
    
    MANTA-3239 marlin agent crash after invalid cueball state transition
    Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/dev/package.json b/dev/package.json
index 46d11e4..ff2b4cb 100644
--- a/dev/package.json
+++ b/dev/package.json
@@ -16,7 +16,7 @@
 		"assert-plus": "^1.0.0",
 		"bunyan": "0.20.0",
 		"catstreams": "0.5.0",
-		"cueball": "2.2.4",
+		"cueball": "2.2.6",
 		"dtrace-provider": "0.6.0",
 		"extsprintf": "^1.3.0",
 		"fuzzstream": "0.0.1",
